mindsdb_native/libs/controllers/functional.py

```diff
@@ -441,6 +441,10 @@ def get_model_data(model_name=None, lmd=None):
                         except Exception:
                             print(f'No column importances found for {icol} !')
 
+            for key in ['train_data_accuracy', 'test_data_accuracy', 'valid_data_accuracy']:
+                if key in lmd:
+                    mao[key] = lmd[key]
+
```

Comment:
Perfect, added all of them in my mindsdb (server) PR so that we are exposing them to the GUI, go ahead and merge